src/main/java/org/javarosa/core/model/IndexedFixtureIndex.kt

```diff
@@ -0,0 +1,7 @@
+package org.javarosa.core.model;
+
+import org.javarosa.core.model.instance.TreeElement
+
+data class IndexedFixtureIndex(val name: String, val base: String, val child: String, val attrs: TreeElement?)
```

Comment:
Can we call this "IndexedFixtureIdentifier" or "IndexedFixtureBase" I think using the same word in two different contexts in an object name ("Index" refers to two different concepts) makes it harder to understand.

Reply:
Also, this class should have a docstring to convey what the class does